test/python/result/test_mitigators.py

```diff
@@ -383,37 +324,33 @@ def test_repeated_qubits_parameter(self, circuits_data):
                 ),
             )
 
-    @data([test_data["test_1"]])
-    @unpack
-    def test_qubits_subset_parameter(self, circuits_data):
+    def test_qubits_subset_parameter(self):
         """Tests mitigation on a subset of the initial set of qubits."""
-        counts_ideal_2 = Counts({"0": 5000, "1": 5000})
-        counts_ideal_6 = Counts({"0": 10000})
 
-        counts_noise = Counts(
-            {"000": 4844, "001": 4962, "100": 56, "101": 65, "011": 37, "010": 35, "110": 1}
-        )
-        CRM = CorrelatedReadoutMitigator(
-            circuits_data["correlated_method_matrix"], qubits=[2, 4, 6]
-        )
-        LRM = LocalReadoutMitigator(circuits_data["local_method_matrices"], qubits=[2, 4, 6])
-        mitigators = [CRM, LRM]
+        shots = 10000
+        assignment_matrices = self.assignment_matrices()
+        mitigators = self.mitigators(assignment_matrices, qubits=[2, 4, 6])
+        circuit, _, _ = self.first_qubit_h_3_circuit()
+        counts_ideal, counts_noise, _ = self.counts_data(circuit, assignment_matrices, shots)
+        counts_ideal_2 = marginal_counts(counts_ideal, [0])
+        counts_ideal_6 = marginal_counts(counts_ideal, [2])
+
         for mitigator in mitigators:
             mitigated_probs_2 = (
                 mitigator.quasi_probabilities(counts_noise, qubits=[2])
                 .nearest_probability_distribution()
-                .binary_probabilities()
+                .binary_probabilities(num_bits=1)
             )
             mitigated_error = self.compare_results(counts_ideal_2, mitigated_probs_2)
             self.assertTrue(
-                mitigated_error < 0.001,
+                mitigated_error < 0.01,
```

Comment:
How come this is necessary?  Looks a bit worrying that it increases by an order of magnitude.

Reply:
Not sure how it got there - the previous code was fine. Fixing.